content/en/serverless/installation/go.md

```diff
@@ -78,9 +78,96 @@ datadog-ci lambda instrument -f my-function -f another-function -r us-east-1 -e
 
 More information and additional parameters can be found in the [Datadog Serverless CLI][2].
 
+
 [1]: https://aws.github.io/aws-sdk-go-v2/docs/getting-started/#get-your-aws-access-keys
 [2]: https://docs.datadoghq.com/serverless/serverless_integrations/cli
 {{% /tab %}}
+{{% tab "Serverless Framework" %}}
+
+The [Datadog Serverless Plugin][1] automatically configures your functions to send metrics, traces, and logs to Datadog through the [Datadog Lambda Extension][2].
+
+To install and configure the Datadog Serverless Plugin, follow these steps:
+
+1. Install the Datadog Serverless Plugin:
+	  ```sh
+    yarn add --dev serverless-plugin-datadog
```

Comment:
Do we really want to use yarn for our documentation instead of npm? I know we like to use yarn here at Datadog, but according to [this survey](https://2021.stateofjs.com/en-US/other-tools) for example only 11% of respondents used it, presumably 90% are using npm.

Reply:
(I posted a question about this in Slack to get input from others)

Reply:
Per Slack convo, let's remove all references to `yarn` and use `npm` instead.

Reply:
(The commands will be slightly different)